hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/AsyncFSWAL.java

```diff
@@ -237,8 +237,8 @@ public AsyncFSWAL(FileSystem fs, Abortable abortable, Path rootDir, String logDi
       ThreadPoolExecutor threadPool =
         new ThreadPoolExecutor(1, 1, 0L,
           TimeUnit.MILLISECONDS, new LinkedBlockingQueue<Runnable>(),
-            new ThreadFactoryBuilder().setNameFormat("AsyncFSWAL-%d-" + rootDir.toString()).
-              setDaemon(true).build());
+          new ThreadFactoryBuilder().setNameFormat("AsyncFSWAL-%d-prefix:" +
```

Comment:
I prefer we still add the rootDir to the name? We can append the prefix after rootDir.toString

Reply:
Thanks @Apache9 , I add the rootDir to the thread name as before.